apps/app/src/app/core/services/settings.service.ts

```diff
@@ -88,22 +88,22 @@ export class SettingsService {
    *
    * @returns True if current applications contains application, false otherwise
    */
-  private hasCurrentApplication(applicationId: Application['_id']): boolean {
-    for (const id of this.currentApplications) {
-      if (
-        id.applicationName === applicationId.applicationName &&
-        id.applicationVersion === applicationId.applicationVersion
-      ) {
-        return true;
-      }
-    }
+  private hasCurrentApplication(applicationId: string): boolean {
```

Comment:
Subnav will be remove in a future PR because there is no more ID concept with proto